internal/pkg/teleport/secret.go

```diff
@@ -35,12 +36,19 @@ func GetConfigFromSecret(ctx context.Context, ctrlClient client.Client, namespac
 		return nil, microerror.Mask(err)
 	}
 
+	if err := ctrlClient.Get(ctx, types.NamespacedName{
+		Name:      key.TeleportBotSecretName,
```

Comment:
Do I understand correctly that `tbot` will create this identity secret which can then be picked up by the operator?

Reply:
Yes, that's correct.